projects/packages/forms/src/blocks/field-textarea/edit.js

```diff
@@ -25,26 +24,21 @@ export default function TextareaFieldEdit( props ) {
 	} );
 
 	const defaultLabel = __( 'Message', 'jetpack-forms' );
+
+	const templateLabel = label ?? '';
 	const template = useMemo( () => {
 		return [
-			[ 'jetpack/label', { label, required, defaultLabel, requiredText } ],
+			[ 'jetpack/label', { label: templateLabel, required, defaultLabel, requiredText } ],
```

Comment:
```suggestion
			[ 'jetpack/label', { label: templateLabel, defaultLabel, requiredText } ],
```

Not introduced here but it looks like there's an obsolete `required` still being passed to the Label block. It should get that from context.